GraphQL/Types.php

```diff
@@ -59,11 +65,13 @@ private function createObjectType($className)
             'name' => (new \ReflectionClass($className))->getShortName(),
             'fields' => function () use ($className) {
                 $classMetadata = $this->entityManager->getClassMetadata($className);
-                $fields = array_reduce($classMetadata->fieldMappings, function ($acc, $mapping) {
+                $fields = array_reduce($classMetadata->fieldMappings, function ($acc, $mapping) use ($classMetadata) {
                     $type = $this->convertFieldMappingToType($mapping);
+                    $fieldName = $mapping['fieldName'];
+                    $excludes = self::EXCLUDE_FIELDS[$classMetadata->name];
```

Comment:
Customer, Member 以外でエラーとなりますのでチェックが必要かと思います。

```suggestion
                    $excludes = array_key_exists($classMetadata->name, self::EXCLUDE_FIELDS) ? self::EXCLUDE_FIELDS[$classMetadata->name] : [];
```